data/sql/updates/pending_db_world/rev_1641245804524923284.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1641245804524923284');
+
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES (5459, 0, 2, 0, 54, 0, 100, 0, 0, 0, 0, 0, 0, 46, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Centipaar Tunneler - On Just Summoned - Move Forward 1 Yards');
```

Comment:
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
(5459, 0, 2, 0, 54, 0, 100, 0, 0, 0, 0, 0, 0, 46, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Centipaar Tunneler - On Just Summoned - Move Forward 1 Yards');

Reply:
Add line Break for readability. One extremely long line does make things a bit problematic on the human eye and other reviewers. We prefer to keep it all on one page.

Reply:
Thanks a lot. Will take it into account next time.

Reply:
Not next time. This time.

Also you always need to a DELETE statement before an INSERT
You can learn more about SQL stuff here https://www.azerothcore.org/wiki/sql-standards :)